src/index.html

```diff
@@ -1,13 +1,263 @@
+        <img
+        src="./images/Natsionalnij-hudozhnij-muzej-Ukrayini-Kiyiv-10-1800x1200.jpg"
+        class="header__photo"
+        alt="Museum"
+        >
+        <div class="header__top">
+          <div class="header__nav-toggler"></div>
+          <a href="#" class="header__logo">
+            NA
+            <br>
+            MU
+          </a>
+        </div>
+        <div class="header__bottom">
+          <h1 class="header__title">ХУДОЖ<span class="header__title--accent">НІЙ</span> МУЗЕЙ</h1>
+          <a href="#" class="header__ticket-buy-link">Квитки</a>
+        </div>
+        <div class="header__right">Події</div>
+      </div>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a href="#gallery" class="header__nav-link">Галерея</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#events" class="header__nav-link">Виставки та події</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#schedule" class="header__nav-link">Відвідування</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#subscription" class="header__nav-link">Стати нашим другом</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Про Нас</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Магазин</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Блог</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#contacts" class="header__nav-link">Контакти</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <section class="events" id="events">
+        <h2 class="events__title">Актуальні події</h2>
+        <article class="events__article">
+          <img
+            src="./images/painting1.jfif"
+            alt="event painting"
+            class="events__image"
+          >
+          <div class="events__wrapper">
+            <div class="events__info">
+              <p class="events__name">ВИСТАВКА</p>
+              <p class="events__date">26.08-29.11.2019</p>
+            </div>
+            <h3 class="events__event-title">Йду і повертаюсь</h3>
+            <p class="events__description">
+              Національний Художній Музей України презентує унікальну частину колекції Градобанку - українське мистецтво другої половини 1980-1995 років.
+            </p>
+          </div>
+        </article>
+        <article class="events__article">
+          <img
+            src="./images/painting2.jfif"
+            alt="event painting"
+            class="events__image"
+          >
+          <div class="events__wrapper">
+            <div class="events__info">
+              <p class="events__name">ВИСТАВКА</p>
```

Comment:
```suggestion
              <p class="events__name">Виставка</p>
```
Better to make all texts lowercase and in css use `text-transform: uppercase`